src/main/kotlin/com/sourcegraph/cody/agent/protocol/ProtocolTextDocument.kt

```diff
@@ -26,15 +28,7 @@ private constructor(
     val contentChanges: List<ProtocolTextDocumentContentChangeEvent>? = null,
     val testing: TestingParams? = null,
 ) {
-
-  init {
```

Comment:
I think that `if` was there for a good reason.
At the very least we should check when/why it was added and if really it is not needed anymore.